src/control/src/espnow_ctrl.c

```diff
@@ -44,11 +44,11 @@ static const char *TAG = "espnow_ctrl";
 
 static espnow_bindlist_t g_bindlist = {0};
 
-int8_t g_rssi;
-uint32_t g_timestamp;
-espnow_ctrl_bind_cb_t g_bind_cb;
-espnow_ctrl_data_cb_t g_data_cb;
-espnow_ctrl_data_raw_cb_t g_data_raw_cb;
+int8_t g_rssi = 0;
+uint32_t g_timestamp = 0;
+espnow_ctrl_bind_cb_t g_bind_cb = NULL;
+espnow_ctrl_data_cb_t g_data_cb = NULL;
+espnow_ctrl_data_raw_cb_t g_data_raw_cb = NULL;
```

Comment:
```cpp
espnow_ctrl_bind_cb_t g_bind_cb = nullptr;
espnow_ctrl_data_cb_t g_data_cb = nullptr;
espnow_ctrl_data_raw_cb_t g_data_raw_cb = nullptr;
```
Are you sure that **NULL** works and should it not set to **nullptr**?

Reply:
Hi @nielsnl68 , nullptr is introduced in C++11. I can understand the need to migrate to nullptr in C++. Could you elaborate why we need nullptr in C here? Are you able to build with nullptr here? For me using nullptr is an undeclared error when building this component. Thanks.